libs/gpu/src/dawn.zig

```diff
@@ -60,6 +60,29 @@ pub const TogglesDeviceDescriptor = extern struct {
     }
 };
 
+pub const TogglesDescriptor = extern struct {
```

Comment:
Many thanks for this PR; could you remove `TogglesDescriptor` for now, though? I want to make sure that we 1:1 reflect what is in Dawn's `webgpu.h`, when we do update later I can add this back - I just want to ensure we're 1:1 with the header API.

Reply:
That makes sense, done! I also went ahead and renamed the MR because there are no new structs in here anymore.